segment_data/traffic_data.py

```diff
@@ -2,25 +2,41 @@
 import requests
 import datetime
 import pytz
+import json
 
 
 STRAVA_ACCESS_TOKEN = os.environ['STRAVA_ACCESS_TOKEN']
 TRAFFIC_URL = "https://www.strava.com/api/v3/segments/%s/all_efforts"
 CALIFORNIA_TIMEZONE = "US/Pacific"
+REDWOOD_SEGMENT_INPUT_FILE = 'redwood_segment_input.json'
 
 
-def get_traffic_count(segment_id, time_parameter):
+def build_segment_id_dict(segment_input_file):
```

Comment:
Yeah, but think about it; what's the difference between:
```
[
 { id: 0, strava_ids: [...] }
 { id: 1, strava_ids: [...] }
]
```

and:

```
{
  0: [ ... ]
  1: [ ... ]
}
```

I would say they are close enough that it's probably not worth this extra function, but it's up to you in the end.